pwiz_tools/Skyline/Util/Extensions/Text.cs

```diff
@@ -629,12 +629,13 @@ public void Initialize(TextReader reader, char separator, bool hasHeaders = true
             }
             for (int i = 0; i < fields.Length; ++i)
             {
-                FieldNames.Add(fields[i].Trim());
-                FieldDict[fields[i]] = i;
+                var trimmed = fields[i].Trim();

```

Comment:
I would name this variable "fieldName" instead of "trimmed".

Reply:
sold!